### ISSUE TITLE
Is there any way to increase the training steps as like Epoch in Tensorflow?

### ISSUE THREAD

**Issue by rajdeinno** [online-ml/river#723]
I need to improve the accuracy of the training. In tensorflow, we have Epoch. Is there any similar iteration configurations in river?

**Comment by MaxHalford** [comment]
Hello. How about just adding a `for` loop on top of your current training routine? Another epoch is "just" an extra pass over a dataset.

**Comment by rajdeinno** [comment]
Hi,

Here's my code.

model = linear_model.LogisticRegression()
dataset = stream.iter_pandas(xtrain, ytrain)

for x, yi in dataset:   
   
    model.learn_one(x, yi)

So, if i have 50 records in dataset, my Epoch is 50? is iteration enough to improve the accuracy?

**Comment by gbolmier** [comment]
I think what Max meant here was smth like:

```python
epoch = 10
for _ in range(epoch):
    for x, yi in dataset:
        model.learn_one(x, yi)
```
